app/policies/submission_answer_policy.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class SubmissionAnswerPolicy < ApplicationPolicy
+  attr_reader :user, :submission_answer
+
+  def initialize(user, submission_answer)
+    super()
+    @user = user
+    @submission_answer = submission_answer
+  end
+
+  def show?
+    user.admin? ||
+      user.mentor? ||
+      user.adviser? ||
+      user.graduated_on? ||
+      submission_answer.practice.completed?(user) ||
```

Comment:
[want]この条件ですが、バグのせいで見れてしまっているので以下とかどうでしょうか。
```suggestion
      submission_answer.practice.product(user)&.checked?
      # 下の条件も削除
```

Reply:
バグ起票していただいているやつですね！
`app/policies/product_policy.rb`と同じ書き方をしたので、一旦コメントアウトで`submission_answer.practice.completed?(user)`を外す旨を書いたうえでご指摘どおりに修正させていただきます！

Reply:
やはりコメントアウトではなく、「提出物がクリアしていること」を先に条件として持ってくるようにしました。
587f7b2015b03d63fd12b8d1b4229c684d096a90

(ないとは思いますが)提出物はクリアでプラクティス修了はしていない、という状況でも見れてしまうので、`practice.completed?(user)`は最後の条件式として残しておきました。

Reply:
> 提出物はクリアでプラクティス修了

そんなシチュエーションが存在したんですね👀
承知しました！